Frontend/ui-driver/src/Screens/HomeScreen/View.purs

```diff
@@ -191,16 +191,15 @@ view push state =
   frameLayout
   [ height MATCH_PARENT
   , width MATCH_PARENT
-  ]([ relativeLayout
+  ]([ PrestoAnim.animationSet [ Anim.fadeIn true ] $ relativeLayout
       [ height MATCH_PARENT
       , width MATCH_PARENT
       , background Color.white900
       , weight 1.0
-      , afterRender
+      , onAnimationEnd
         (\action -> do
           _ <- push action
           _ <- JB.setFCMToken push $ SetToken
-          _ <- JB.showMap (EHC.getNewIDWithTag "DriverTrackingHomeScreenMap") (enableCurrentLocation state) "satellite" (17.0) push ShowMap
           _ <- JB.getCurrentPosition push CurrentLocation
```

Comment:
can we move this function as well to `onAnimationEnd`

Reply:
@S-PRAVINKUMAR can you also check here that after map is rendered marker is upserted at user's current position in first run

Reply:
> can we move this function as well to `onAnimationEnd`

@ashishsingh101  this is already in the onAnimationEnd only

Reply:
> @S-PRAVINKUMAR can you also check here that after map is rendered marker is upserted at user's current position in first run

@ashishsingh101 yes it is present at drivers current position.